platform_plugin_forum_email_notifier/static/html/forum_notifier.html

```diff
@@ -1,15 +1,35 @@
 {% load i18n %}
 
 <div class="email-notifier-instructor-wrapper">
-    {% trans "Forum email notifications preferences" %}
+    <div>
+        <p class="title-list">{% trans "In this section you can set your email notification preferences for the discussion forum. The available options are:" %}</p>
+        <ul class="list-container">
+            <li>{% trans "None: I don't want to receive any email notifications." %}</li>
+            <li>{% trans "All posts: I want to receive email notifications for all posts." %}</li>
+            <li>
+                {% trans "Only posts I'm following: I want to receive email notifications for posts I'm following." %}
+            </li>
+            <li>
+                {% trans "All posts (Daily digest): I want to receive email notifications for all posts in a daily digest." %}
+            </li>
+            <li>
+                {% trans "All posts (Weekly digest): I want to receive email notifications for all posts in a weekly digest." %}
+            </li>
+        </ul>
+    </div>
 
-    <select class="email-notifications-select">
+    <p><b>{% trans "These notifications are only sent to the instructor of the course." %}</b></p>
```

Comment:
Can you remove this part?

The notification are also sent to students but students doesn't have email preferences as of now